src/content/getting-started/prepare-your-provider-account/aws/_index.md

```diff
@@ -2,11 +2,216 @@
+
+- [giantswarm-${INSTALLATION_NAME}-capa-controller-policy](https://github.com/giantswarm/giantswarm-aws-account-prerequisites/raw/master/capa-controller-role/capa-controller-policy.json)
+- [giantswarm-${INSTALLATION_NAME}-crossplane-policy](https://github.com/giantswarm/giantswarm-aws-account-prerequisites/raw/master/capa-controller-role/crossplane-policy.json)
+- [giantswarm-${INSTALLATION_NAME}-dns-controller-policy](https://github.com/giantswarm/giantswarm-aws-account-prerequisites/raw/master/capa-controller-role/dns-controller-policy.json)
+- [giantswarm-${INSTALLATION_NAME}-eks-controller-policy](https://github.com/giantswarm/giantswarm-aws-account-prerequisites/raw/master/capa-controller-role/eks-controller-policy.json)
+- [giantswarm-${INSTALLATION_NAME}-iam-controller-policy](https://github.com/giantswarm/giantswarm-aws-account-prerequisites/raw/master/capa-controller-role/iam-controller-policy.json)
+- [giantswarm-${INSTALLATION_NAME}-irsa-operator-policy](https://github.com/giantswarm/giantswarm-aws-account-prerequisites/raw/master/capa-controller-role/irsa-operator-policy.json)
+- [giantswarm-${INSTALLATION_NAME}-mc-bootstrap-policy](https://github.com/giantswarm/giantswarm-aws-account-prerequisites/raw/master/capa-controller-role/mc-bootstrap-policy.json)
+- [giantswarm-${INSTALLATION_NAME}-network-topology-operator-policy](https://github.com/giantswarm/giantswarm-aws-account-prerequisites/raw/master/capa-controller-role/network-topology-operator-policy.json)
+- [giantswarm-${INSTALLATION_NAME}-resolver-rules-operator-policy](https://github.com/giantswarm/giantswarm-aws-account-prerequisites/raw/master/capa-controller-role/resolver-rules-operator-policy.json)
+
+__Warning__: Remember to replace the `INSTALLATION_NAME` placeholder with the name of your installation when filling the policy name.
+
+![AWS IAM console: Create policy](aws-roles-create-policy.png)
+
+__Note__: All policy names contain the installation name to make easier identifying the policies and avoid conflicts with other installations running within the same account.
+
+#### 2. Create the role metadata {#iam-capa-controller-role-basic}
+
+Now move to the IAM [Roles](https://console.aws.amazon.com/iam/home#/roles) subsection, and hit the **Create role** button. In the following screen, when asked to _Select type of trusted entity_ chose _AWS account_.
+
+![AWS IAM console: Create role](aws-roles-create-role.png)
+
+The **Account ID** you enter is the ID of the AWS account where management cluster is running. Could it be the current account in case you running all your infrastructure in a single account or different one if you are using separate accounts for management and workload clusters.
```

Comment:
```suggestion
The __Account ID__ you enter is the ID of the AWS account where the management cluster is running. Could it be the current account in case you are running all your infrastructure in a single account or a different one if you are using separate accounts for management and workload clusters?
```